Sources/SmokeDynamoDB/QueryInput+forSortKeyCondition.swift

```diff
@@ -25,7 +25,8 @@ extension QueryInput {
                                                                  sortKeyCondition: AttributeCondition?,
                                                                  limit: Int?,
                                                                  scanIndexForward: Bool,
-                                                                 exclusiveStartKey: String?) throws
+                                                                 exclusiveStartKey: String?,
+                                                                 consistentRead: Bool?) throws
```

Comment:
Do we have to write like `consistentRead: Bool? = nil` ?

Reply:
We already set default value at the public interface leve. So I think we don't need to set default value here again.